Jenkinsfile

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env groovy
+
+/**
+ * Build Contiki Applications
+ *
+ */
+
+
+node('docker && imgtec') {
+    def docker_image
+    docker_image = docker.image "imgtec/creator-builder:latest" // TODO for now using the creator one
+    docker_image.inside {
+        stage('Prepare') {
+            checkout([$class: 'GitSCM',
+                userRemoteConfigs: scm.userRemoteConfigs,
+                branches: scm.branches,
+                doGenerateSubmoduleConfigurations: scm.doGenerateSubmoduleConfigurations,
+                submoduleCfg: scm.submoduleCfg,
+                browser: scm.browser,
+                gitTool: scm.gitTool,
+                extensions: scm.extensions + [
+                    [$class: 'CleanCheckout'],
+                    [$class: 'PruneStaleBranch'],
+                ],
+            ])
+        }
+        stage('Build') {
+            sh 'git submodule init "dev/ca8210"'
+            sh 'git submodule update'
+            sh 'cd examples/hello-world && make TARGET=mikro-e USE_CA8210=1 CHANNEL=26 PAN_ID=0xabcd USE_SERIAL_PADS=1'
```

Comment:
Add the make arguments as parameters in properties section

properties([
    buildDiscarder(logRotator(numToKeepStr: '5')),
    parameters([
        stringParam(defaultValue: '26', description: 'Set 6LoWPAN Channel', name: "CHANNEL"),
        stringParam(defaultValue: ''0xacbd', description: 'Set 6LoWPAN PAN ID', name: 'PAN_ID'),
        stringParam(defaultValue: '1', description: 'Set Serial port to use for console', name: 'PADS'),
    ] + feedParams)
])